webpack/assets/javascripts/react_app/components/DualList/DualList.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DualListControlled as PfDualList,
+  Col,
+  ControlLabel,
+  FormGroup,
+} from 'patternfly-react';
+import { arrangeItemsBySelectedIDs } from './helpers';
+import { bindMethods, noop } from '../../common/helpers';
+import './dual-list.scss';
+
+class DualList extends React.Component {
+  constructor(props) {
+    super(props);
+    bindMethods(this, ['handleInit', 'handleChange']);
+  }
+
+  handleInit({ right: { items } }) {
+    const { initialUpdate, id } = this.props;
+    const selectedItems = items.map(({ label, value }) => ({ label, value }));
+    initialUpdate(selectedItems, id);
+  }
+
+  handleChange({ right: { items } }) {
+    const { onChange, id } = this.props;
+    const selectedItems = items.map(({ label, value }) => ({ label, value }));
+    onChange(selectedItems, id);
+  }
+
+  render() {
+    const { inputProps, items, selectedIDs, label, id } = this.props;
+    const { selectedList, unselectedlist } = arrangeItemsBySelectedIDs(
+      items,
+      selectedIDs
+    );
+
+    return (
+      <FormGroup id={id}>
```

Comment:
The #6677 is merged, would it make sense to use it as a wrapper for the component?
I would love to see it usable from the rails helper.